sdk/core/Azure.Core/perf/DynamicObjectBenchmark.cs

```diff
@@ -12,19 +12,24 @@
 
 namespace Azure.Core.Perf
 {
-    [SimpleJob(RuntimeMoniker.NetCoreApp31, baseline: true)]
-    [SimpleJob(RuntimeMoniker.Net462)]
-    [SimpleJob(RuntimeMoniker.Net60)]
+    [SimpleJob(RuntimeMoniker.Net80)]
+    [MemoryDiagnoser]
     public class DynamicObjectBenchmark
     {
-        private static string _fileName = Path.Combine(Path.Combine(Directory.GetParent(Assembly.GetExecutingAssembly().Location).FullName, "TestData", "JsonFormattedString.json"));
+        private string _fileName;
         private JsonDocument _jsonDocument;
         private ModelWithBinaryData _modelWithBinaryData;
         private ModelWithObject _modelWithObject;
 
+        private static readonly string FileName = Path.Combine(
```

Comment:
```suggestion
        private static readonly string _fileName = Path.Combine(
```
nit - in the repo we usually try to have private fields use this naming style